### ISSUE TITLE
Button to roll hp when leveling up.

### ISSUE THREAD

**Issue by joesinghaus** [Karlinator/roll20-character-sheets#11]
Some classes get extra hp per level. How should we handle this – a query asking for "extra hp per hit dice"? Other suggestions?

**Comment by PanzerUXI** [comment]
I agree that makes the most sense. On the HP result roll, we should have our italicized Arial demystify how it rolled below the result - like. "Rolled 3d6+6+6, +2/Con*3, +2/Warrior*3" or something?
